tests/testthat/test-plotting.R

```diff
@@ -7,46 +7,46 @@ rm(list = ls())
 # expect_silents only work with this logger turned off; only alerts with warnings
 loggerOptions <- futile.logger::logger.options()
 if (!identical(loggerOptions, list())){
-  origLogThreshold <- loggerOptions[[1]][['threshold']]
+    origLogThreshold <- loggerOptions[[1]][['threshold']]
 } else {
-  origLogThreshold <- futile.logger::INFO
+    origLogThreshold <- futile.logger::INFO
 }
 futile.logger::flog.threshold(0)
 
 test_that('node coloring by discrete and continuous', {
-  b <- FunctionReporter$new()
-  b$set_package('baseballstats'
-                , pkg_path = system.file('baseballstats'
-                                         , package = "pkgnet"
-                                         , lib.loc = Sys.getenv('PKGNET_TEST_LIB')
-                                         )
-                )
-  b$.__enclos_env__$private$set_plot_node_color_scheme(
-      field = "coverageRatio"
-      , pallete = c("red", "green")
-  )
-  
-  expect_silent({
-      
+    b <- FunctionReporter$new()
+    b$set_package('baseballstats'
```

Comment:
are these changes related to the intent of the PR? Or just other things you noticed?

Reply:
I think what happened was:
- I opened this file because I was intending to change something about it
- I fixed the indentation to match our style guide
- Then I don't remember if I actually changed anything else. From a quick skim I don't think I ended up changing anything else.